scaladoc/src/dotty/tools/scaladoc/tasty/ClassLikeSupport.scala

```diff
@@ -64,14 +64,18 @@ trait ClassLikeSupport:
     signatureOnly: Boolean = false,
     modifiers: Seq[Modifier] = classDef.symbol.getExtraModifiers(),
   ): Member =
-    def unpackTreeToClassDef(tree: Tree): ClassDef = tree match
-      case tree: ClassDef => tree
-      case TypeDef(_, tbt: TypeBoundsTree) => unpackTreeToClassDef(tbt.tpe.typeSymbol.tree)
-      case TypeDef(_, tt: TypeTree) => unpackTreeToClassDef(tt.tpe.typeSymbol.tree)
-      case c: Apply =>
-        c.symbol.owner.tree.symbol.tree match
+    def unpackTreeToClassDef(tree: Tree): ClassDef =
+      def unpackApply(a: Apply) =
+        a.symbol.owner.tree.symbol.tree match
```

Comment:
Is the `.symbol.tree` needed?
I think that it should be the case that `∀(s: Symbol). s.tree ≡ s.tree.symbol.tree`

Reply:
I suppose you're right. I will remove it.